packages/fx-core/src/component/driver/aad/utility/buildAadManifest.ts

```diff
@@ -97,3 +99,57 @@ async function loadManifest(
     }
   }
 }
+
+function validateManifest(manifest: AADManifest): void {
+  const missingProperties: string[] = [];
+
+  if (isNullOrUndefined(manifest.addIns)) {
```

Comment:
Some fields can be null. Here we just check properties that cannot be null in the manifest.

Reply:
You can put the required value into a tuple or an array, and then traverse it.
And there are too many if conditional judgments here. If there is an addition or deletion later, it may be clearer to use an array

Reply:
I used another method to fix the bug and the code is no longer needed